examples/https/server-cert.pem

```diff
@@ -1,22 +1,22 @@
 -----BEGIN CERTIFICATE-----
-MIIDozCCAougAwIBAgIUNmVva8tI/hqzZAB7FZzPmNr/Q0owDQYJKoZIhvcNAQEL
```

Comment:
It would be great to have all cert updates in a different PR so this can move forward faster and unblock everyone creating PRs and getting errors because of it.

Reply:
Certs removed.